packages/croncat-sdk-tasks/src/types.rs

```diff
@@ -343,7 +343,7 @@ pub struct SlotIdsResponse {
     pub block_ids: Vec<u64>,
 }
 
-#[derive(Debug, PartialEq)]
+#[cw_serde]
```

Comment:
```suggestion
#[allow(clippy::derive_partial_eq_without_eq)]
#[derive(Debug, PartialEq)]
```
Will work too, if you want to min-max wasm size

Reply:
I saw that - i think we can do optimization path separate across all contracts